releases/v21.2.0.md

```diff
@@ -0,0 +1,127 @@
+Core       | **Third-party tool support**   | [Alembic](../v21.2/alembic.html) and [Sequelize](../v21.2/build-a-nodejs-app-with-cockroachdb-sequelize.html) are now fully supported. PgBouncer, an external connection pooler for Postgres. [Alembic](../v21.2/alembic.html) is a schema migration tool developed for use with SQLAlchemy, the Python data access framework. Other third-party tools such as Apache Airflow also leverage Alembic underneath for schema management. [Sequelize](../v21.2/build-a-nodejs-app-with-cockroachdb-sequelize.html) is a Javascript object-relational mapper (ORM) that allows you to perform database operations without leaving your programming language.
+Core       | **Admission control**          | A new [admission control system](../v21.2/architecture/admission-control.html) has been added. CockroachDB implements this optional admission control system to maintain cluster performance and availability when some nodes experience high load. Admission control is disabled by default.<br><br>Additionally, an [**Overload** dashboard](../v21.2/ui-overload-dashboard.html) has been added, which lets you monitor the performance of the parts of your cluster relevant to the cluster's [admission control system](../v21.2/architecture/admission-control.html). This includes CPU usage, the runnable goroutines waiting per CPU, the health of the persistent stores, and the performance of admission control system when it is enabled.
+Core       | **Transaction guardrails**     | Transaction guardrails have been added to CockroachDB to improve production stability. These can help prevent cluster unavailability and protect the cluster against multiple developer workloads. <!-- add link when Stephanie publishes PR: ../v21.2/transactions.html#limit-the-number-of-rows-written-or-read-in-a-transaction -->
+Core       | **Expression indexes**         | [Indexes on expressions](../v21.2/expression-indexes.html) can now be created. These indexes speed up queries that filter on the result of that expression, and are especially useful for indexing only a specific field of a `JSON` object.
+Core       | **Correlated CTEs**            | [Correlated common table expressions](../v21.2/common-table-expressions.html#correlated-common-table-expressions) (CTEs) are now supported in CockroachDB. A correlated CTE is when a common table expression is contained in a subquery and the CTE references columns defined outside of the subquery.
+Core       | **Persistent statement and transaction statistics** | Statistics information on the [**Statements**](../v21.2/ui-statements-page.html) and [**Transactions**](../v21.2/ui-transactions-page.html) pages within the DB Console can now be persisted for longer than one hour.
```

Comment:
Need to mentioned unused index stats cc @kevin-v-ngo

Reply:
Perhaps something along the lines of 

Index usage statistics

Index usage statistics are now supported for CockroachDB to help [identify unused indexes](https://www.cockroachlabs.com/docs/v21.2/performance-recipes-solutions?filters=indexusage#fix-slow-writes) causing unnecessary performance overhead for your workload. Index read statistics are available in the [crdb_internal schema](https://www.cockroachlabs.com/docs/dev/crdb-internal.html#data-exposed-by-crdb_internal) for programmatic access using SQL.